src/elements/content-sidebar/SidebarNavSign.tsx

```diff
@@ -22,10 +24,28 @@ export type Props = PlainButtonProps & {
 
 export const PlaceholderTooltip = ({ children }: { children: React.ReactNode }) => children;
 
-export function SidebarNavSign({ intl, status, targetingApi, ...rest }: Props) {
+export function SidebarNavSign({ blockedReason, intl, isDisabled, status, targetingApi, ...rest }: Props) {
```

Comment:
Can we remove `isDisabled` now?

Reply:
Left it here in case it gets passed in for whatever reason and it needs to override the calculated value

Reply:
It's unused at the moment, so we should probably remove it. If it ever is needed, it should be covered by `...rest`.

Reply:
ah that's right